### PR TITLE
Semi-automatic update of generated content

### PR DESCRIPTION
**Pull request by taylorbot**
This PR has been created automatically by the 'Update generated content' Github workflow
to keep

- Changes and releases
- CLI versions
- Cluster Apps
- CRD pages
- Website header and footer

up-to-date.

This PR is opened or updated every weekday morning. On Fridays, it is merged by 'merge-autogenerated-content' workflow. In case of being triggered manually you need to merge it manually too in order to have the content published.

### CODE REVIEW

**Review by AndiDog — APPROVED** [review]

File: src/content/changes/workload-cluster-releases-vsphere/releases/vsphere-28.0.0.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+# Generated by scripts/aggregate-changelogs. WARNING: Manual edits to this files will be overwritten.
+aliases:
+- /changes/tenant-cluster-releases-vsphere/releases/vsphere-vsphere-28.0.0/
+changes_categories:
+- Workload cluster releases for VSPHERE
+changes_entry:
+  repository: giantswarm/releases
+  url: https://github.com/giantswarm/releases/tree/master/vsphere/v28.0.0
+  version: sphere-28.0.0
```
Comment (AndiDog) [inline src/content/changes/workload-cluster-releases-vsphere/releases/vsphere-28.0.0.md:10]: @pipo02mix It trims off `v` which isn't desired here 😆

**Review by AndiDog — DISMISSED** [review]

File: src/content/changes/workload-cluster-releases-vsphere/releases/vsphere-28.0.0.md (hunk 1 above)
Comment (pipo02mix) [reply src/content/changes/workload-cluster-releases-vsphere/releases/vsphere-28.0.0.md:10]: hmm, that is a good catch. I will take a look now

Comment (pipo02mix) [inline src/content/changes/workload-cluster-releases-vsphere/releases/vsphere-28.0.0.md:10]:
```suggestion
  version: vsphere-28.0.0
```

**Review by pipo02mix — APPROVED** [review]

**Comment by pipo02mix** [comment]
@AndiDog I fixed the version name manually and I will check the fix in the script in the next PR

**Review by pipo02mix — APPROVED** [review]

**Pull request merged**